_includes/sidebar.html

```diff
@@ -147,7 +147,7 @@
     </li>
   </div>
   <div class="sidebarbuttoncontainer">
-    <li class="folder topfolder">
+    <div class="folder topfolder">
```

Comment:
Dto.

Reply:
With the current structure, this should be in its own `li`—or does this not work?

Reply:
fixed